checks/check-app-health.go

```diff
@@ -0,0 +1,59 @@
+package checks
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"net/http"
+	"io/ioutil"
+    "strconv"
+    "log"
+
+	maps "github.com/ashwanthkumar/golang-utils/maps"
+	"github.com/gambol99/go-marathon"
+)
+
+type AppHealth struct{}
+
+func (h *AppHealth) Name() string {
+	return "apphealth"
+}
+
+// Checks App health status of App Endpoint by reading router.hosts 
+//  and Check for 200 status code
+func (h *AppHealth) Check(app marathon.Application) AppCheck {
+	host := maps.GetString(app.Labels, "router.hosts", "notDefined")
+    host_path  := app.HealthChecks[0].Path
+	host_url := strings.Join([]string{"http://", host, host_path}, "")
+    result := Pass
+    message := fmt.Sprintf("HTTP Status OK!!")
+
+	resp, err := http.Get(host_url)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
`log.Fatal` kills the go process right? We don't want that. Please log the message and return the status as Failed.

Reply:
Logging the message now and returning a Warning with the error status. Do we create a new flag for non-app failures?